app/abilities/sac_cas/person_ability.rb

```diff
@@ -14,10 +14,20 @@ module SacCas::PersonAbility
     on(Person) do
       permission(:read_all_people).may(:read_all_people, :show).everybody
       class_side(:create_households).if_sac_mitarbeiter
+      permission(:any).may(:set_sac_family_main_person).if_all_household_member_writable
+    end
+
+    def if_sac_mitarbeiter
+      SacCas::SAC_MITARBEITER_ROLES.any? { |r| role_type?(r) }
     end
-  end
 
-  def if_sac_mitarbeiter
-    SacCas::SAC_MITARBEITER_ROLES.any? { |r| role_type?(r) }
+    def if_all_household_member_writable
+      return false if person.household_people.count.clamp(0, 1).zero?
```

Comment:
Brauchts das `clamp` hier? 
`person.household_people.blank?` sollte doch denselben Zweck erfüllen und finde ich leserlicher :thinking:

Reply:
oder evtl. noch besser `.empty?`

Reply:
habe empty? genommen. Das macht im hintergrund exists?
[Present? Vs Any? Vs Exists? - The Lean Software Boutique](https://www.ombulabs.com/blog/benchmark/performance/rails/present-vs-any-vs-exists.html)